scripts/generate_weights.sh

```diff
@@ -29,7 +29,7 @@ bench_run() {
 bench_run xpallet_assets            ./xpallets/assets/src/weights.rs
 bench_run xpallet_assets_registrar  ./xpallets/assets-registrar/src/weights.rs
 bench_run xpallet_dex_spot          ./xpallets/dex/spot/src/weights.rs
-# bench_run xpallet_gateway_bitcoin   ./xpallets/gateway/bitcoin/src/weights.rs
+bench_run xpallet_gateway_bitcoin   ./xpallets/gateway/bitcoin/src/weights.rs
 #bench_run xpallet_gateway_common    ./xpallets/gateway/common/src/weights.rs
```

Comment:
Are you going to fix `common` benchmarks in the follow-up PR?

Reply:
Yeah, I will try, but there is no guarantee that it will be fixed before the estimated time of 2.0 :(